gbpservice/nfp/tests/unit/nfp/configurator/test_firewall.py

```diff
@@ -0,0 +1,887 @@
+import requests
+import unittest
+import mock
```

Comment:
This file is too big. Lets split into multiple as per the following split.
(1) Configurator test cases
(2) Generic config agent test cases
(3) Firewall agent test cases
(4) Firewall driver test cases.